[NEW] survey_record_generation (#149)
OCA/survey: survey_record_generation/readme/CONTEXT.rst

> In several cases we want to use survey application to create records. 

Reviewer: ```suggestion
In several cases we want to use the Survey application to create records. 
```

> Website form application give the same possibility but the design is integrated on the web site. 

Reviewer: ```suggestion
The Website Form application gives the same possibility but the design is integrated in the web site. 
```

> Futhermore this module add other functionnalities like "duplicate" check, or creation of several interconnected records

Reviewer: ```suggestion
Furthermore this module adds other functionalities like checking for "duplicates", or creating several interconnected records
```

> Typical use case : Information request form

Reviewer: ```suggestion
A typical use case is an Information request form:
```
